debs/SPECS/4.2.6/wazuh-agent/debian/changelog

```diff
@@ -1,3 +1,9 @@
+wazuh-agent (4.2.6-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Mon, 25 Mar 2022 16:47:07 +0000
```

Comment:
Correct the date.

Reply:
Done 6318a8a96caf59fd477451b2802b4f8c620b3eb6